ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh

```diff
@@ -133,13 +133,26 @@ EOF
     for EXTENSION in "${EXTENSIONS_TO_DISABLE[@]}"; do
         EXTENSION_ENABLED=$(run_sql -A -t -c "SELECT EXISTS(SELECT 1 FROM pg_extension WHERE extname = '${EXTENSION}');")
         if [ "$EXTENSION_ENABLED" = "t" ]; then
+            echo "Exporting dependent objects for extension ${EXTENSION}, if they exist"
+            EXPORT_QUERY=$(cat << EOF
+    SELECT pg_get_functiondef(objid) || ';' AS function_definition
```

Comment:
Ideally we would have tests for this query

Reply:
Hmm does this capture multiple levels of dependencies?

e.g. view -> table -> function -> generated column uses plv8

Reply:
I'm guessing dropping the ext would cause the entire chain to be dropped?

Reply:
Yep, pretty much. Currently looking into a more involved way to detect these